stm32-modules/common/STM32F303/STM32F303RETx_FLASH.ld

```diff
@@ -67,8 +67,18 @@ SECTIONS
     . = ALIGN(4);
   } >FLASH
 
+  /* Contains firmware authentication information */
+  .integrity_info BLOCK(0x200) :
+  {
+    . = ALIGN(4);
+    /* placeholder for CRC */
+    LONG(0);
```

Comment:
I _think_ you  could do something like 
```
.integrity_info BLOCK(0x200) :
  {
    . = ALIGN(4);
    _integrity_crc = .;
    LONG(0);
    _integrity_length=.;
    LONG(0);
} >FLASH
```

and then reference those symbols with extern in the actual code

Reply:
in fact, you might be able to get the linker to put the overall length in there

Reply:
One issue is that this is the linker for the runtime app, not the startup app; but adding this to the startup linker as a NOLOAD should do the trick I think?